compiler/rustc_parse/src/parser/attr.rs

```diff
@@ -452,7 +454,7 @@ impl<'a> Parser<'a> {
     /// ```ebnf
     /// MetaItemInner = UNSUFFIXED_LIT | MetaItem ;
     /// ```
-    fn parse_meta_item_inner(&mut self) -> PResult<'a, ast::NestedMetaItem> {
+    pub fn parse_meta_item_inner(&mut self) -> PResult<'a, ast::NestedMetaItem> {
```

Comment:
An aside: `NestedMetaItem` would be better named as `MetaItemInner`, to match the grammar.

Reply:
Agree, at first I was bit confused about `MetaItemInner` vs `NestedMetaItem`, until realizing that they are the same.

Reply:
Opened https://github.com/rust-lang/rust/issues/131087 to track this renaming.